bin/chrootBuddy.sh

```diff
@@ -57,7 +57,7 @@ OPTIONS:
 
 EXAMPLE:
 	sudo ./chrootBuddy.sh  --verbose --name enet --release stable --prefix /srv/chroot --arch amd64 --extra cuda --packages enet
-	sudo ./chrootBuddy.sh  --verbose --name ssd  --release stable --prefix /srv/chroot --arch amd64 --extra "cuda extra" --packages "opencv ssd"
+	sudo ./chrootBuddy.sh  --verbose --name ssd  --release stable --prefix /srv/chroot --arch amd64 --extra "cuda extra" --packages "opencv ssd enet"
```

Comment:
--name ssd
then no need to add 'enet' package
or please change name option to 'epoc'
Un peu de l'enc... de mouche je l'accorde ... mais je trouve moins confusing ;)